packages/nimble-components/src/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -156,6 +156,45 @@ export class RichTextEditorPageObject {
             .map(el => el.textContent || '');
     }
 
+    public isFooterVisible(): boolean {
+        const footerSection = this.richTextEditorElement.shadowRoot!.querySelector(
+            '.footer-section'
+        )!;
```

Comment:
minor: Query selection part can go into a separate page object function `getFooter()` that can be reused in all higher order functions?

Reply:
Updated